jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/roles/CaseRolesViewImpl.java

```diff
@@ -84,35 +104,43 @@ public void removeAllRoles() {
     }
 
     @Override
-    public void enableNewRoleAssignments() {
-        removeCSSClass(userAddButton, "hidden");
-    }
-
-    @Override
-    public void disableNewRoleAssignments() {
-        addCSSClass(userAddButton, "hidden");
-    }
-
-    @Override
-    public void setUserAddCommand(final Command command) {
-        this.userAddCommand = command;
+    public void setFilterCommand(Command command) {
+        this.filterCommand = command;
     }
 
-    @Override
-    public void addUser(final String userName, final String roleName, final CaseRolesPresenter.CaseRoleAction... actions) {
-        addRoleView(userName, roleName, "pficon-user", actions);
+    public void addOwnerAssignment(String roleName, String user) {
+        final CaseRoleItemView roleItemView = provider.get();
+        roleItemView.setRoleName(roleName);
+        roleItemView.setOwnerAssignment(user);
+        allElementsList.add(roleItemView);
     }
 
-    @Override
-    public void addGroup(final String groupName, final String roleName, final CaseRolesPresenter.CaseRoleAction... actions) {
-        addRoleView(groupName, roleName, "pficon-users", actions);
+    public String getFilterValue() {
+        if (isNullOrEmpty(filterSelect.getValue())) {
+            return "All";
+        }
+        return filterSelect.getValue();
```

Comment:
Maybe would make more sense to move the logic of processing _filterSelect_ value to presenter and leave just a simple getter method in the view.